packages/renderer/src/lib/preferences/LineWithLinks.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+export let lineText: string;
+
+interface Segment {
+  text: string;
+  href?: string;
+}
+
+function createLink(text: string): Segment {
```

Comment:
Maybe it's overkill in this case, but I'm wondering if we should just allow markdown? I think we're going to need it in other places (e.g. issue #1989) and would allow other basic styling instead of custom parsing/styling.

Reply:
Correct. I mixed up api with ui at first. The file was here. https://github.com/containers/podman-desktop/pull/2175/commits
/468d92800080cbabe488ad4d027c5479ca3d348a#diff-8e4ce8f5e8aaf1925d5dca6f6253666e3795a3abda6ff6496e790a16fadb752e